src/styles/index.scss

```diff
@@ -1,3 +1,104 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+}
+
 body {
   margin: 0;
 }
+
+.product-cart {
+  box-sizing: border-box;
+  padding-inline: 16px;
+  width: 200px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.product__name--code {
```

Comment:
The class name '.product__name--code' does not match the class used in the HTML file, which is 'product-cart__name-code'. Ensure the class names are consistent between the HTML and SCSS files to apply the styles correctly.